sdk_container/src/third_party/coreos-overlay/app-emulation/docker/docker-20.10.24-r1.ebuild

```diff
@@ -152,6 +153,12 @@ pkg_setup() {
 		"
 	fi
 
+	if kernel_is lt 6 1; then
+		CONFIG_CHECK+="
+			~MEMCG_SWAP
+		"
+	fi
+
```

Comment:
Nice catch.

Ideally, however, we should first create a commit for syncing with Gentoo, (like ee672bcdb9cd414480b48389f5fd07a6148a0125) and then create another commit for Flatcar changes (like 742c507270c1d80629c1a4d4297a5cd93c37d765), so that all other Kernel-related changes could be updated as well.

Reply:
Right, I should have followed that. Will do.

Reply:
Synced properly. Needed to pull in a new version of app-emulation/docker-proxy.

Reply:
The Docker part looks good